stdlib/public/runtime/Exclusivity.cpp

```diff
@@ -94,14 +77,52 @@ static void reportExclusivityConflict(ExclusivityFlags oldAction, void *oldPC,
     return;
   }
 
-  fprintf(stderr,
-          "Simultaneous accesses to 0x%lx, but modification requires exclusive "
-          "access.\n",
-          (uintptr_t)pointer);
-  printConflictDetails(getAccessName(oldAction), oldPC,
-                       getAccessName(getAccessAction(newFlags)), newPC);
+  constexpr unsigned maxMessageLength = 100;
+  constexpr unsigned maxAccessDescriptionLength = 50;
+  char message[maxMessageLength];
+  snprintf(message, sizeof(message),
+           "Simultaneous accesses to 0x%lx, but modification requires "
+           "exclusive access",
+           (uintptr_t)pointer);
+  fprintf(stderr, "%s.\n", message);
+
+  char oldAccess[maxAccessDescriptionLength];
+  snprintf(oldAccess, sizeof(oldAccess),
+           "Previous access (a %s) started at", getAccessName(oldAction));
+  fprintf(stderr, "%s ", oldAccess);
+  if (oldPC) {
+    dumpStackTraceEntry(0, oldPC, /*shortOutput=*/true);
+    fprintf(stderr, " (0x%lx).\n", (uintptr_t)oldPC);
```

Comment:
I'm not trying to change the text output with this patch, so I think this is okay.  Notice the `/*shortOutput=*/true`, so the whole line looks like:
```
Previous access (a modification) started at module`your_function+0xaa (0x1000a0cd).
```